docs/source/core_concepts/er_model.rst

```diff
@@ -0,0 +1,83 @@
+Entity-relationship model
+=========================
+
+Each Hyperledger Iroha peer has a state, called "World State View",
+which is represented by a set of entities and relations between them.
+To explain you more which entities exist in the system and what are the relations,
+this sections includes ER diagram and an explanation for its components.
```

Comment:
https://www.italki.com/question/267722
Looks like, it will be correct to say "an explanation of" in this case

Reply:
🙆